### PR TITLE
Section 1-8

### PR DESCRIPTION
**Pull request by RaymondMichael**
Master committee PR

### CODE REVIEW

**Comment by wrrobin** [comment]
In Section 2, we have an overview of the OpenSHMEM routines, where we have not referred any team specific routines, non-blocking routines, or the new signaling operations, such as put-signal. Do we need to update this section? Or have we decided that we do not need to refer them here?

**Comment by RaymondMichael** [comment]
My personal opinion is that it isn't needed. If you write up something though, we can discuss it.

**Comment by nspark** [comment]
:+1: These changes look good so far!

File: content/execution_model.tex [hunk 1]
```diff
@@ -71,6 +71,17 @@ \subsection{Invoking OpenSHMEM Operations}
...
 \ac{PE} $i$ results in undefined behavior.
 
+Buffers provided to \openshmem routines are \emph{in-use} until the
+corresponding \openshmem operation has completed at the calling \ac{PE}.
+Updates to a buffer that is in-use, including updates performed through locally
+and remotely issued \openshmem operations, result in undefined behavior.
+Similarly, reads from a buffer that is in-use are allowed only when the buffer
+was provided as a \CTYPE{const}-qualified argument to the \openshmem routine for
+which it is in-use.  Otherwise, the behavior is undefined. Exceptions are made for
+buffers that are in-use by \acp{AMOs}, as described in
```
Comment (nspark) [inline content/execution_model.tex]:
```suggestion
buffers that are in-use by \acp{AMO}, as described in
```

File: content/execution_model.tex [hunk 2]
```diff
@@ -71,6 +71,17 @@ \subsection{Invoking OpenSHMEM Operations}
...
+Section~\ref{subsec:amo_guarantees}}. For information regarding the completion
+of \openshmem operations, see Section~\ref{sec:memory_order}.
```
Comment (nspark) [inline content/execution_model.tex]:
```suggestion
of \openshmem operations, see Section~\ref{subsec:memory_order}.
```

**Comment by jdinan** [comment]
These changes have been merged to master via the v1.5rc2 PR, with the exception of 18b2c4ae which is under consideration as a solution to #313.

**Comment by jdinan** [comment]
Closing section committee PR, since we have transitioned to FRM.

**Pull request closed without merging**